[DOC] #23 and #24 removed network policies and dangerous capabilities from lists of tests

agentpoyo opened the pull request:
## Pre-submission checklist:

This is to remove `network_policies` and `dangerous_capabilities` from list of tests.

Ref #: #23 and #24 and #38

lixuna reviewed (changes requested):
please see https://github.com/cncf/cnf-certification/issues/23#issuecomment-1346975651 and https://github.com/cncf/cnf-certification/issues/24#issuecomment-1346970429

lixuna commented on docs/CNFCertification-1.1-beta.md:
```diff
@@ -0,0 +1,439 @@
+# CNF Certification List of Tests - v1.0-beta
```
```suggestion
# CNF Certification List of Tests - v1.1-beta
```

lixuna commented on docs/CNFCertification-1.1-beta.md:
```diff
@@ -0,0 +1,439 @@
+
+To learn why these tests were written, see the [RATIONALE.md](https://github.com/cncf/cnf-testsuite/blob/main/RATIONALE.md)
+
+### Types of Tests (Currently 57 total for the Certification)
```
@agentpoyo has the number of tests changed after removing 2? were any other tests added? Please review this section for accuracy

agentpoyo replied:
The count in the list is now 55. I think I saw 54 tests when I ran `cert` to validate https://github.com/cncf/cnf-certification/issues/38 so I'll get confirmation on actual number.

agentpoyo replied:
There are 56 tests. I think we might of miscounted to get to 57 in the original doc. I'm going to confirm that there is indeed 56 labeled as `cert` in the code now. @lixuna

agentpoyo replied:
https://github.com/cncf/cnf-certification/issues/38#issuecomment-1352047218

agentpoyo replied:
According to the v1 list of tests spreadsheet https://docs.google.com/spreadsheets/d/1xOxrXzexUaV-hLnPCT7NgcF6pWVWuVUrsDMXFIc8XXc/edit#gid=836418693

We did not have rolling_update as a v1 test, it's not marked as a `cert` test currently and does not run, but it is listed in the List of Tests so I will get that removed.

agentpoyo commented:
@lixuna 

I think this is ready now. Confirmed 55 tests. Updated the totals at top in the types and removed `rolling_update` which was in the list of tests but was not an actual `cert` test according to our original v1 spreadsheet and was not marked in the points.yml as a `cert` test.

lixuna reviewed (approved):
thank you, lgtm
